posts/7-common-errors-to-check-when-debugging-airflow-dag.md

```diff
@@ -1,223 +1,203 @@
 
-If tasks are getting bottlenecked and your concurrency configurations all look alright, it might just be that your Scheduler is underpowered or that your deployment could use another worker. If you're using Astronomer, we generally recommend 5 AU as the default minimum for the Scheduler and 10 AU for your Celery workers if you have them.
+### Try Scaling up your Scheduler or adding a Worker.
 
-Whether or not you beef up your current resources or add an extra worker is very much dependent on your use case, but we generally recommend the following:
+If tasks are getting bottlenecked and your concurrency configurations look alright, it might just be that your Scheduler is underpowered or that your Deployment could use another Celery Worker, assuming you're using the [Celery Executor](https://airflow.apache.org/docs/apache-airflow/2.0.1/executor/celery.html). If you're running on Astronomer, we generally recommend 5 AU as the default minimum for the Scheduler and 10 AU for Celery Workers.
 
-   *  If you're running a relatively high number of light tasks across DAGs and at a relatively high frequency, you're likely better off having 2 or 3 "light" workers to spread out the work
+Whether or not you scale your current resources or add an extra Celery Worker depends on your use case, but we generally recommend the following:
 
-   * If you're running fewer but heavier tasks at a lower frequency, you're likely better off with a single but "heavier" worker that can more efficiently execute those tasks
+   * If you're running a relatively high number of light tasks across DAGs and at a relatively high frequency, you're likely better off having 2 or 3 "light" workers to spread out the work.
+   * If you're running fewer but heavier tasks at a lower frequency, you're likely better off with a single but "heavier" worker that can more efficiently execute those tasks.
 
-For more info on the differences between Executors, check out our [Airflow Executors: Explained](https://www.astronomer.io/guides/airflow-executors-explained/) Guide.
+For more information on the differences between Executors, we recommend reading [Airflow Executors: Explained](https://www.astronomer.io/guides/airflow-executors-explained/).
 
-## 6. You're Missing Logs
+## 6. You're Missing Logs.
 
-This one comes up quite a bit too. Generally speaking, logs fail to show up because of a process that died on one of your workers.
+Generally speaking, logs fail to show up because of a process that died on your Scheduler or one or more of your Celery Workers.
 
-You might see something like the following:
+If you're missing logs, you might see something like the following:
```

Comment:
Where would you see this?

Reply:
RT ^ Would you see this in place of the logs?

Reply:
Good callout! Added:

> If you're missing logs, you might see something like this under "Log by attempts" in the Airflow UI: